src/lib/components/course/tabs/course-instructors.svelte

```diff
@@ -29,8 +29,12 @@
     </CardHeader>
     <CardContent>
         {#await instructors}
+            {$inspect(instructors)}
             <p class="text-center">Loading...</p>
         {:then instructors}
+            {#if instructors.length == 0}
```

Comment:
```suggestion
            {#if instructors.length === 0}
```
Comparison == may cause unexpected type coercion